features/nanostack/nanostack-hal-mbed-cmsis-rtos/ns_event_loop.c

```diff
@@ -2,18 +2,20 @@
  * Copyright (c) 2016 ARM Limited, All Rights Reserved
```

Comment:
As we are changing this file, also license should fixed (can we make sure have proper license headers in nanostack) ?